springboard/tests/test_views.py

```diff
@@ -12,7 +14,7 @@ def setUp(self):
         self.workspace = self.mk_workspace()
         self.config = testing.setUp(settings={
             'unicore.repos_dir': self.working_dir,
-            'unicore.content_repo_url': self.workspace.working_dir,
+            'unicore.content_repo_urls': self.workspace.working_dir,
```

Comment:
Pyramid doesn't seems to do anything in addition to `ConfigParser`. So you just end up with a string. `pyramid_jinja2` uses [this](https://github.com/Pylons/pyramid_jinja2/blob/master/pyramid_jinja2/settings.py#L43) to parse extensions. `unicore-cms` parses featured languages like [this](https://github.com/universalcore/unicore-cms/blob/develop/cms/views/cms_views.py#L54). Both are quite icky.